### PR TITLE
Added spell for woofi dex on base

### PR DESCRIPTION
**Pull request by Jason4276**
Added trades data for woofi dex on base.

### CODE REVIEW

**Comment by Jason4276** [comment]
I have read the CLA Document and I hereby sign the CLA.

**Comment by jeff-dude** [comment]
> I have read the CLA Document and I hereby sign the CLA.

please copy paste exactly, the `.` will likely cause the bot to fail

**Comment by Jason4276** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Jason4276** [comment]
Recheck

File: macros/models/_sector/dex/woofi_compatible_trades.sql
Comment (Hosuke) [inline macros/models/_sector/dex/woofi_compatible_trades.sql:1]: We may use `generic_spot_compatible_trades` macro for woofi:
https://github.com/duneanalytics/spellbook/blob/main/macros/models/_sector/dex/generic_spot_compatible_trades.sql

**Comment by jeff-dude** [comment]
seems this was auto-merged when #5144 was merged, likely due to latest force push a few minutes ago?

there were no file changes, so not sure if you need to reopen new PR for woofi addition

**Pull request merged**